microsoft-edge/extensions-chromium/developer-guide/sidebar.md

```diff
@@ -153,6 +153,8 @@ chrome.tabs.onUpdated.addListener(async (tabId, info, tab) => {
 
 When a user switches to a tab or site for which the sidebar is not enabled, the sidebar is hidden.
 
+The sidebar is not automatically displayed again when the user switches to a tab in which the sidebar was previously open.<!-- todo: if behavior changes, update or remove statement -->
```

Comment:
Is there a way for the extension developer to persist the display of the sidebar in the tab?

rendered section:
https://review.learn.microsoft.com/en-us/microsoft-edge/extensions-chromium/developer-guide/sidebar?branch=pr-en-us-3328#enable-a-sidebar-for-a-specific-site-only

Reply:
status:
https://github.com/microsoft/MicrosoftEdge-Extensions/issues/142